pyproject.toml

```diff
@@ -20,6 +20,7 @@ cryptography = ">=37.0.4"
 packaging = "^23.1"
 ruff = "^0.0.270"
```

Comment:
lol I just noticed now ruff is under normal dependencies.

Reply:
resolving since this is not introduced here

Reply:
https://github.com/quipucords/qpc/pull/261 nice